test/client.js

```diff
@@ -1151,28 +1151,29 @@ describe('client API', function() {
   });
 
   describe('Multiple output proposals', function() {
-    it('should create, get, sign, and broadcast proposal', function(done) {
+    var toAddress = 'n2TBMPzPECGUfcT2EByiTJ12TPZkhN2mN5';
+    var opts = {
+      type: 'multiple_outputs',
+      message: 'hello',
+      outputs: [{
+        amount: 10000,
+        toAddress: toAddress,
+        message: 'world',
+      }, {
+        amount: 20000,
+        toAddress: toAddress,
+        message: null,
+      }, {
+        amount: 30000,
+        toAddress: toAddress,
+      }]
+    };
+    function doit(opts, doNotVerifyPayPro, doBroadcast, done) {
       helpers.createAndJoinWallet(clients, 1, 1, function(w) {
```

Comment:
Could you please move creation of the wallet & address outside of this function so it only gets called once at the start of the test?